releasenotes/notes/44152.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+releaseNotes:
+  - |
+    **Fixed** operator skip reconcile for iop resources named starting with `installed-state`, and now it only relies on annotation `install.istio.io/ignoreReconcile`.
```

Not sure if this sounds better.
```suggestion
    **Removed** operator skip reconcile for iop resources with names starting with `installed-state`. It now relies solely on the annotation `install.istio.io/ignoreReconcile`.
```

---

Revised together with the change above, thanks.